### DISCUSSION TITLE
Mention Code owners in the PR body

### DISCUSSION THREAD

**Discussion by nicoklaus** [renovatebot/renovate#31373] [Request Help]
### What would you like help with?

I would like help with my configuration

### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, tell us which platform (GitHub, GitLab, etc) and which version of Renovate.

GitLab

### Please tell us more about your question or problem

Hello Together,

The community edition of GitLab don't allow multiple reviewers or assignees on an issue. Therefore, it would be helpful if you could use the extracted code owners from renovate to mention in the PR body and also trigger a notification. So, my question would be, if there is a variable which contains the code owners of a repository and could be used via the (prBodyNotes)[https://docs.renovatebot.com/configuration-options/#prbodynotes].

Something like:

```yaml
{
  "prBodyNotes": ["{{#if isMajor}Please check {{reviewers}} {{/if}}"]
}
```

Thank you.

### Logs (if relevant)

<details><summary>Logs</summary>

```

Replace this text with your logs, between the starting and ending triple backticks

```

</details>

**Comment by rarkins** [comment]
I don't think any such variable exists already but you're welcome to look into what it would take a submit a PR. `{{reviewers}}` is not the right term though, it should specifically be something like `{{codeOwners}}` and may need some further formatting if it's not a list of `@username`s
